src/dev/tsconfig.dev.json

```diff
@@ -6,6 +6,9 @@
     "moduleResolution": "node",
     "target": "es5"
   },
-
+  "angularCompilerOptions": {
+    "fullTemplateTypeCheck": true,
+    "strictInjectionParameters": true
+  },
```

Comment:
This looks good. As a reminder for myself it would be good to see if we can add this to the website docs as well.

Reply:
I think this doesn't need any mention. The current Angular CLI add this to tsconfig.json when generating a new project.